lisa/src/main/java/it/unive/lisa/program/cfg/Parameter.java

```diff
@@ -133,4 +143,22 @@ public String toString() {
 	public CodeLocation getLocation() {
 		return location;
 	}
+
+	/**
+	 * Yields the annotations of this parameter.
+	 * 
+	 * @return the annotations of this parameter
+	 */
+	public Annotations getAnnotations() {
+		return annotations;
+	}
+
+	/**
+	 * Adds an annotations to this parameter.
+	 * 
+	 * @param ann the annotation to be added
+	 */
+	public void addAnnotation(Annotation ann) {
```

Comment:
In some classes we have `addAnnotation`, in some other we have `setAnnotations`. We should unify them